### PR TITLE
Rahulsandhu3

### PR DESCRIPTION
**Pull request by rahulsandhu2555**

### CODE REVIEW

File: www/src/components/containers/country/components/accordian/Accordion.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
```
Comment (Pranav2612000) [inline www/src/components/containers/country/components/accordian/Accordion.tsx:1]: Use the Accordian component from commons/Accordion

File: www/src/components/containers/country/components/infoCard/infoCard.module.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	position: absolute;
+	display: none;
+}
+
+/* Show the checkmark when checked */
+.container input:checked ~ .checkmark:after {
+	display: block;
+}
+
+/* Style the checkmark/indicator */
```
Comment (Pranav2612000) [inline www/src/components/containers/country/components/infoCard/infoCard.module.css:69]: Are these the same styles as the checkmark component?

File: www/src/components/common/Button/index.tsx [hunk 3]
```diff
@@ -6,6 +6,7 @@ type IButtonProps = {
 	onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
 	type: 'button' | 'submit' | 'reset';
 	className?: string;
+	style?: 'active' | 'nonActive';
```
[the remark this answers is not in the document]
Comment (rahulsandhu2555) [reply www/src/components/common/Button/index.tsx:9]: variant, which you updated to light and dark

File: www/src/components/common/Popover/index.tsx [hunk 4]
```diff
@@ -44,6 +44,10 @@ const Popover = ({
 		setMounted(true);
 	}, [selector]);
 
+	useEffect(() => {
+		console.log('rerendering popover');
```
[the remark this answers is not in the document]
Comment (rahulsandhu2555) [reply www/src/components/common/Popover/index.tsx:48]: you added this  one

File: www/src/components/common/Popover/popover.module.css [hunk 5]
```diff
@@ -24,7 +24,16 @@
 .traveller {
 	height: 85%;
 }
-
+.popupdiv_small {
```
[the remark this answers is not in the document]
Comment (rahulsandhu2555) [reply www/src/components/common/Popover/popover.module.css:27]: this was for the small size popover, do we not need it now? as you asked to create  a small size popover style

File: www/src/components/common/Popover/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Pranav2612000) [reply www/src/components/common/Popover/index.tsx]: Possible. Nikal de par abhi.

File: www/src/components/common/Button/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Pranav2612000) [reply www/src/components/common/Button/index.tsx]: Right. So we don't need this. Right?

File: www/src/components/common/Popover/popover.module.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (Pranav2612000) [reply www/src/components/common/Popover/popover.module.css]: There is already a class for small popover. Can that be reused?

**Pull request merged**